data/discretize/discretize.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+def discretize(location, grid):
+    return tuple(int(np.digitize(l, g)) for l, g in zip(location, grid))
```

Comment:
1. Is the grid necessarily to be `int`? If not, the test is not exhausted. Otherwise should modify the description to specify the grid is `int`.
2. Take a look at this [https://numpy.org/doc/stable/reference/generated/numpy.digitize.html](https://numpy.org/doc/stable/reference/generated/numpy.digitize.html). Maybe should specify `right` argument. Consider the edge case that the location falls exactly on grid point, and the grid array is increasing/decreasing.